addons/binding/org.openhab.binding.paradoxalarm/src/main/java/org/openhab/binding/paradoxalarm/internal/handlers/ParadoxPartitionHandler.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.paradoxalarm.internal.handlers;
+
+import static org.openhab.binding.paradoxalarm.internal.handlers.ParadoxAlarmBindingConstants.*;
+
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNull;
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.openhab.binding.paradoxalarm.internal.exceptions.ParadoxBindingException;
+import org.openhab.binding.paradoxalarm.internal.model.ParadoxPanel;
+import org.openhab.binding.paradoxalarm.internal.model.Partition;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link ParadoxPartitionHandler} Handler that updates states of paradox partitions from the cache.
+ *
+ * @author Konstantin_Polihronov - Initial contribution
+ */
+public class ParadoxPartitionHandler extends EntityBaseHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(ParadoxPartitionHandler.class);
```

Comment:
make this static

Reply:
loggers are not made static in openHAB

Reply:
All loggers will be made non-static as per Hilbrand's proposal even though I'm not sure it's good when creating multiple handlers for different zones which pretty much look the same.